### PR TITLE
Ringcentral updates

### PR DESCRIPTION
**Pull request by yslpn**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/r/ringcentral.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 cask "ringcentral" do
-  arch arm: "-arm"
+  arch arm: "-arm64", intel: ""
```
Comment (bevanjkay) [inline Casks/r/ringcentral.rb:2]: Intel can be omitted as it is automatically an empty string. 

```suggestion
  arch arm: "-arm64"

**Review by bevanjkay — DISMISSED** [review]
Based on the presence of the electron builder feed Id like us to check if there's a versioned url available.

File: Casks/r/ringcentral.rb (hunk 1 above)
Comment (yslpn) [reply Casks/r/ringcentral.rb]: Okay, thanks)

**Comment by yslpn** [comment]
> Based on the presence of the electron builder feed Id like us to check if there's a versioned url available.

Yesterday I tried to find a link with the version, but I couldn’t find it. Thanks for the help!

**Comment by yslpn** [comment]
@bevanjkay is everything okay?

**Review by bevanjkay — APPROVED** [review]

**Comment by bevanjkay** [comment]
Yea, looks good to me. Needs a review from another maintainer, since I pushed the last commit.

**Review by krehel — APPROVED** [review]

**Pull request merged**